addons/pos_self_order/static/src/app/pages/order_history_page/order_history_page.js

```diff
@@ -23,9 +23,7 @@ export class OrdersHistoryPage extends Component {
     }
 
     getPrice(line) {
-        return this.selfOrder.config.iface_tax_included
-            ? line.price_subtotal_incl
-            : line.price_subtotal;
+        return this.selfOrder.config.iface_tax_included ? line.displayPrice : line.displayPriceUnit;
```

Comment:
```suggestion
        return  line.displayPrice;
```
The displayPrice method already takes iface_tax_included into account.